src/features/pages/deriv-ez/hero/data.tsx

```diff
@@ -6,24 +6,27 @@ import DerivEzLogo from 'images/svg/deriv-ez/deriv-ez-logo.svg'
 export const hero_images = {
```

Comment:
Don't we need typescript here

Reply:
nope, it's always ReactElement

Reply:
then we can change the data file name to ts and remove React import here